v2/pkg/common/constants.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package common
+
+// =================== Config variable constants ===================
+
+const (
+	// #nosec
+	ClientSecretVar      = "AZURE_CLIENT_SECRET"
+	SubscriptionIDVar    = "AZURE_SUBSCRIPTION_ID"
+	TenantIDVar          = "AZURE_TENANT_ID"
+	ClientIDVar          = "AZURE_CLIENT_ID"
+	ClientCertificateVar = "AZURE_CLIENT_CERTIFICATE"
+	// #nosec
+	ClientCertificatePasswordVar = "AZURE_CLIENT_CERTIFICATE_PASSWORD"
+	TargetNamespacesVar          = "AZURE_TARGET_NAMESPACES"
+	OperatorModeVar              = "AZURE_OPERATOR_MODE"
+	SyncPeriodVar                = "AZURE_SYNC_PERIOD"
+	ResourceManagerEndpointVar   = "AZURE_RESOURCE_MANAGER_ENDPOINT"
+	ResourceManagerAudienceVar   = "AZURE_RESOURCE_MANAGER_AUDIENCE"
+	AzureAuthorityHostVar        = "AZURE_AUTHORITY_HOST"
+	PodNamespaceVar              = "POD_NAMESPACE"
```

Comment:
We should remove the `Var` suffix from all of these, it's kinda implied.

Might pay to put a line or two of documentation on each one, maybe even a link to our docs.